Automattic/wp-calypso
# Sites Management Page: Add Jetpack Stats column

Fixes #68626

## Proposed Changes

Four specific changes:
1. Adds a 'Jetpack Stats' column to the Sites page that displays the 48 hour sparkline.
2. Only loads the sparkline component when the row is in view, to avoid unnecessary expensive API requests.
3. Introduces an opt-in "noise" loading effect for the sparkline component.
4. Tweaks 'Status' column CSS to prevent the progress donut from disappearing.

### Stats column

<img width="812" alt="image" src="https://user-images.githubusercontent.com/36432/194437225-f7e07134-8b96-4600-bfe0-42fd60a01127.png">

### Loading effect


https://user-images.githubusercontent.com/36432/194438358-7dfe5dcb-b335-4670-87f1-4b1bbe0123e1.mp4



## Testing Instructions

1. Navigate to `/sites`.
2. Verify 'Jetpack Stats' column appears as expected.
